lib/clog/clog.go

```diff
@@ -197,6 +204,16 @@ func SetError(ctx context.Context, err error) error {
 
 	ctxLogger.SetError(err)
 
+	// add stack trace if available
+	if serr, ok := err.(stackTracer); ok {
+		st := serr.StackTrace()
```

Comment:
Should we limit the stack trace at all, or would we prefer to dump the whole thing?

Reply:
As you say, you are avoiding turning on error tracking, and are instead just adding more bytes to the lots, but this may add a lot of bytes of data.